build.gradle

```diff
@@ -108,6 +108,8 @@ subprojects {
         grpcVersion = '1.20.0'
         autoValueVersion = '1.6.2'
         opentracingVersion = '0.33.0'
+        protobufVersion = '3.7.1'
```

Comment:
+1.  Not all the artifacts will use `protobufVersion` (i.e. the API itself), but this way we hopefully don't have to re-declare this version again and again ;)